pkg/cloudprovider/providers/vsphere/vsphere.go

```diff
@@ -758,7 +808,7 @@ func (vs *VSphere) DisksAreAttached(volPaths []string, nodeName k8stypes.NodeNam
 		if err != nil {
 			return nil, err
 		}
-		vm, err := vs.getVMByName(ctx, nodeName)
+		vm, err := vs.getVMFromNodeName(ctx, nodeName)
```

Comment:
This logic has been changed. Please get latest from master.

Reply:
I checked master. We need this logic for multi-vc.